addons/markers/functions/fnc_movePFH.sqf

```diff
@@ -0,0 +1,34 @@
+/*
+ * Author: chris579
+ * When the marker is being moved.
+ *
+ * Arguments:
+ * 0: Marker data <ARRAY>
+ * 1: PFH ID <NUMBER>
+ *
+ * Return Value:
+ * None
+ *
+ * Example:
+ * [[MARKER], 5] call ace_markers_fnc_movePFH
+ *
+ * Public: No
+ */
+#include "script_component.hpp"
+
+params ["_args", "_idPFH"];
+_args params ["_marker"];
+
+if (isNull (findDisplay 12 displayCtrl 51) || !(player getVariable [QGVAR(moveInProgress), false])) exitWith {
```

Comment:
He's meaning you should use lazy evaluation.
```
if ( (statement) || { another statement })
```
Arma doesn't lazy evaluate if statements, so both sides of `or` will always execute if the subsequent checks aren't wrapped in `{ }`